bin/main.rb

```diff
@@ -0,0 +1 @@
+puts "Hello World!"
```

Comment:
Let's make the `main.rb` file executable.

Reminder from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)

> Make your main.rb file displaying “Hello world!” when you use bin/main.rb as a bash command.

Reply:
Thank you, will make the appropriate changes and resubmit for code review.